src/agent/rustjail/src/container.rs

```diff
@@ -1378,15 +1378,13 @@ async fn join_namespaces(
         info!(logger, "notify child parent ready to run prestart hook!");
         read_async(pipe_r).await?;
 
-        info!(logger, "get ready to run prestart hook!");
```

Comment:
Why delete this line?

Reply:
Sorry, I thought the two logs before and after have expressed the same meaning. Fixed.